collections/nemo_nlp/nemo_nlp/data/datasets/bert_pretraining.py

```diff
@@ -158,66 +155,127 @@ def get_document(filepath, line):
 
                 # Read line, remove newline, and decode as UTF8
                 doc_text = f.readline()[:-1].decode("utf-8", errors="ignore")
-                document = [self.tokenizer.token_to_id("[CLS]")]
-                document.extend(self.tokenizer.text_to_ids(doc_text))
-                document.append(self.tokenizer.token_to_id("[SEP]"))
+                document = self.tokenizer.text_to_ids(doc_text)
 
-            assert len(document) >= self.max_seq_length
             return document
 
-        a_document = get_document(a_filename, a_line)
+        def match_target_seq_length(document, target_seq_length, filename,
+                                    line_idx, sentence_indices):
+            # match the target seq_length
+            for _ in range(10):
```

Comment:
why is this 10?

Reply:
just followed similarly to the original BERT implementation:
https://github.com/google-research/bert/blob/master/create_pretraining_data.py#L279

10 is an arbitrary choice for now searching the data to match the target_seq_length.